website/views.py

```diff
@@ -339,96 +405,115 @@ def new_question(request):
                     context['tutorial'] = tutorial
                     context['form'] = form
                     context['toolbox'] = toolbox
-                    return render(request, 'website/templates/new-question.html', context)
+                    return render(request,
+                                  'website/templates/new-question.html',
+                                  context)
                 else:
                     pass
                 question.sub_category = ""
                 question.save()
-            question.title = cleaned_data['title']
+            #question.title = cleaned_data['title']
```

Comment:
Please make sure that this code is not required and purge the code base of such evil